example/spec/loadTest/ingestLoad_queueGranSFCompleted.js

```diff
@@ -0,0 +1,278 @@
+      inputMeta
+    );
+
+    queueGranulesExecutionArns.push(workflowExecution.executionArn);
+    totalInputPayloads.push(inputPayload);
+
+    if (workflowExecution.status === 'completed') {
+      console.log(greenConsoleLog(), `\n___ Execution of Workflow ${nthWorkflow} completed with success status ___`);
+    } else {
+      console.log(redConsoleLog(), `\n___ Execution of Workflow ${nthWorkflow} Failed ___\n ${JSON.stringify(workflowExecution)}\n`);
+    }
+
+    const lambdaOutput = await new LambdaStep().getStepOutput(
+      workflowExecution.executionArn,
+      'QueueGranules'
+    );
+    queueGranulesChildExecutionArns.push(lambdaOutput.payload);
+
+    if (lambdaOutput.payload.running.length === granuleCountPerWorkflow) {
+      colorConsoleLog = greenConsoleLog();
+    } else {
+      colorConsoleLog = redConsoleLog();
+      console.log(colorConsoleLog, '\n___ lambdaOutputPayload ExecutionArns ___');
+      lambdaOutput.payload.running.map((runningArn) =>
+        console.log(`${JSON.stringify(runningArn)}`));
+    }
+    console.log(colorConsoleLog,
+      `\n___ Lambda Output of Workflow ${nthWorkflow} has ${lambdaOutput.payload.running.length}/${granuleCountPerWorkflow} of expected arns ___`);
+
+    completedGranules = [];
+    incompletedGranules = [];
+
+    const expectedValues = ['completed'];
+
+    await Promise.all(
+      inputPayload.granules.map(async (granule) => {
+        try {
+          const record = await waitForApiStatus(
+            getGranule,
+            {
+              prefix: config.stackName,
+              granuleId: granule.granuleId,
+            },
+            expectedValues
+          );
+          completedGranules.push(record);
+          totalGranulesCompleted.push(record);
+        } catch (error) {
+          incompletedGranules.push({ granuleId: granule.granuleId, err: error });
+        }
+      })
+    );
+  } catch (error) {
+    beforeAllFailed = `===== batchGranulesProcessing() failed =====\n ${error}`;
+    // throw new Error(beforeAllFailed);
+  } finally {
+    console.log(greenConsoleLog(),
+      `\n___ ${completedGranules.length}/${inputPayload.granules.length} Granules completed by workflow ${nthWorkflow} ___`);
+
+    if (completedGranules.length !== inputPayload.granules.length) {
+      console.log(redConsoleLog(),
+        `\n___ ${incompletedGranules.length}/${inputPayload.granules.length} Granules incompleted by workflow ${nthWorkflow} ___`);
+
+      incompletedGranules.map((granuleError) =>
+        console.log(redConsoleLog(),
+          `\n${JSON.stringify(granuleError.granuleId)} - ${granuleError.err}`));
+    }
+
+    console.log(blueConsoleLog(), `\n===== Clean up S3TestDataFolder: ${testDataFolder} =====`);
+    await deleteFolder(config.bucket, testDataFolder);
+  }
+};
+
+describe('The Granule Ingest Load Test ', () => {
+  beforeAll(async () => {
+    if (!Number.isFinite(Number(granuleCountPerWorkflow)) || granuleCountPerWorkflow > 425) {
```

Comment:
We should store `425` as a variable to spell out what it represents.

Reply:
Also, is `!Number.isFinite(Number(granuleCountPerWorkflow))` the same as `Number(granuleCountPerWorkflow) >= 0`? Are we trying to account for negative numbers?

Reply:
Assigned 425 to a meaningful variable name. 
Number.isFinite checks the number is neither positive infinity, negative Infinity, nor NaN